ligo/src/helper/unpack.mligo

```diff
@@ -7,7 +7,16 @@
 #include "../types.mligo"
 
 // ------------------------------
-// Unpack Helpers
+// Safe Unpack Helpers
+// ------------------------------
+
+let unpack_transfer_type_list_safe (key_name, p : string * ((string, bytes) map)) : (transfer_type list) option =
+  match Map.find_opt key_name p with
+  | None -> (None : (transfer_type list) option)
+  | Some b -> (Bytes.unpack b : (transfer_type list) option)
```

Comment:
This too I'd suggest to move in `registryDAO.mligo` as it is particular to its implementation.